api/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const server = express();
+server.use(express.json());
+const db = require('../database/dbConfig')
+
+server.get('/', (req, res) => {
+    res.status(200).json({ api: 'up' });
+});
+
+server.post('/create-user',(req,res) => {
+    const user = req.body;
+    db('users').insert(user)
+    .then(userId => {
+        res.status(200).json(userId)
+    })
+    .catch(err => {
+        res.status(500).send(err)
+    })
+})
+
+server.delete('/delete-user/:id', (req,res) => {
+    const { id } = req.params;
+
+    db('users').where(id).del()
```

Comment:
You need to change this line of code to `db("users").where({id: id}).del()` or else it will throw an error and make one of the tests fail. Better to pass id as an object with key value pair to be safer.